toolkit/geo-compose/src/main/java/com/arcgismaps/toolkit/geocompose/samples/GeoViewDocSamples.kt

```diff
@@ -0,0 +1,121 @@
+package com.arcgismaps.toolkit.geocompose.samples
+
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Modifier
+import com.arcgismaps.Color
+import com.arcgismaps.data.ServiceFeatureTable
+import com.arcgismaps.geometry.Point
+import com.arcgismaps.geometry.SpatialReference
+import com.arcgismaps.mapping.ArcGISMap
+import com.arcgismaps.mapping.ArcGISScene
+import com.arcgismaps.mapping.ArcGISTiledElevationSource
+import com.arcgismaps.mapping.BasemapStyle
+import com.arcgismaps.mapping.Surface
+import com.arcgismaps.mapping.Viewpoint
+import com.arcgismaps.mapping.layers.FeatureLayer
+import com.arcgismaps.mapping.symbology.SimpleLineSymbol
+import com.arcgismaps.mapping.symbology.SimpleLineSymbolStyle
+import com.arcgismaps.mapping.symbology.SimpleRenderer
+import com.arcgismaps.mapping.view.Camera
+import com.arcgismaps.toolkit.geocompose.MapView
+import com.arcgismaps.toolkit.geocompose.SceneView
+
+/**
+ * @suppress Suppress this function from being indexed in the KDoc
+ */
+@Composable
+public fun SceneViewSample() {
+    // Display a SceneView with an elevation surface and an initial viewpoint
+
+    // add base surface for elevation data
+    val elevationSource = ArcGISTiledElevationSource(
+        uri = "https://elevation3d.arcgis.com/arcgis/rest/services/WorldElevation3D/Terrain3D/ImageServer"
+    )
+    val surface = Surface().apply {
```

Comment:
Just for consistency:
```suggestion
    )
    
    val surface = Surface().apply {
```